src/components/WhoWeAre/Departments.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Masonry from 'react-masonry-css';
+import Accordion from 'react-bootstrap/Accordion';
+import bios from './bios';
+import Team from './Team';
+import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { colors } from '../../theme/styleVars';
+import { media } from 'styled-bootstrap-grid';
```

Comment:
[nit] Minor thing, but let's keep external imports above our local imports. The order should be:
1. React and React-related imports
2. External library imports
3. Local imports
4. Local image or asset imports